src/site/content/en/blog/wakelock/index.md

```diff
@@ -83,11 +85,25 @@ Of course, there are plenty of other use cases:
 
 ## Using the Wake Lock API {: #use }
 
-The Wake Lock API is currently in development and is only available in Chrome
-behind a flag. To experiment with the Wake Lock API, enable the
-`#enable-experimental-web-platform-features` flag in `chrome://flags`.
+### Enabling support during the origin trial phase
+Starting in Chrome 79, the Wake Lock API will be available as an origin trial in Chrome.
+The origin trial is expected to end in Chrome 81.
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### Enabling via chrome://flags
+To experiment with the Wake Lock API locally, without an origin trial token,
+enable the `#experimental-web-platform-features` flag in `chrome://flags`.
+
+### Demo
 
 Check out the [Wake Lock demo][demo] and [demo source][demo-source].
+Notice how the wake lock gets automatically released when you switch tabs
+or multitask away from the browser.
```

Comment:
"multitask away" doesn't sound idiomatic to my ear. "navigate away"?

Reply:
I actually mean switch to a new app, not navigate to a new page.

Reply:
"switch to a different app" sounds good to me :) I'll do that.